### PR TITLE
Added function to determine if application is in production environment

### PR DESCRIPTION
**Pull request by KnysakPatryk**
A similar function for development environment exists. I believe that to be more consistent, there should be a similar function for the production environment.

### CODE REVIEW

File: src/Illuminate/Foundation/Application.php [hunk 1]
```diff
@@ -460,6 +460,16 @@ public function isLocal()
...
     }
+	
+	/**
+     * Determine if application is in production environment.
+     *
+     * @return bool
+     */
+    public function isProduction()
+    {
+        return $this['env'] == 'production';
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Application.php]: what if the env is "prod"?

File: src/Illuminate/Foundation/Application.php [hunk 2]
```diff
@@ -460,6 +460,16 @@ public function isLocal()
     {
         return $this['env'] == 'local';
     }
+	
+	/**
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Application.php]: cs

**Comment by KnysakPatryk** [comment]
@GrahamCampbell thanks for suggestions

File: src/Illuminate/Foundation/Application.php [hunk 3]
```diff
@@ -462,6 +462,16 @@ public function isLocal()
     }
 
     /**
+     * Determine if application is in local production.
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Application.php]: how can it be in "local production"?

Comment (KnysakPatryk) [inline src/Illuminate/Foundation/Application.php]: @GrahamCampbell oh, sorry :(

**Pull request closed without merging**